src/AppBundle/Entity/Comment.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="symfony_demo_comment")
+ *
+ * Defines the properties of the Comment entity to represent the blog comments.
+ * See http://symfony.com/doc/current/book/doctrine.html#creating-an-entity-class
+ *
+ * Tip: if you have an existing database, you can generate these entity class automatically.
+ * See http://symfony.com/doc/current/cookbook/doctrine/reverse_engineering.html
+ *
+ * @author Ryan Weaver <weaverryan@gmail.com>
+ * @author Javier Eguiluz <javier.eguiluz@gmail.com>
+ */
+class Comment
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Post", inversedBy="comments")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $post;
+
+    /**
+     * @ORM\Column(type="text")
+     * @Assert\NotBlank(message="comment.blank")
+     * @Assert\Length(
+     *     min = "5",
+     *     minMessage = "comment.too_short",
+     *     max = "10000",
+     *     maxMessage = "comment.too_long"
+     * )
+     */
+    private $content;
+
+    /**
+     * @ORM\Column(type="string")
+     * @Assert\Email()
+     */
+    private $authorEmail;
+
+    /**
+     * @ORM\Column(type="datetime")
+     * @Assert\DateTime()
+     */
+    private $publishedAt;
+
+    public function __construct()
+    {
+        $this->publishedAt = new \DateTime();
+    }
+
+    /**
+     * @Assert\IsTrue(message = "comment.is_spam")
+     */
```

Comment:
__[development][code_sniffer]__
* Missing @return tag in function comment

<!-- Comment by [SideCI](https://sideci.com) -->